sys/include/net/gnrc/sixlowpan/config.h

```diff
@@ -95,7 +95,7 @@ extern "C" {
  * normally).
  */
 #ifdef DOXYGEN
-#define GNRC_SIXLOWPAN_FRAG_RBUF_DO_NOT_OVERRIDE
+#define CONFIG_GNRC_SIXLOWPAN_FRAG_RBUF_DO_NOT_OVERRIDE
```

Comment:
In the documentation of this macro `GNRC_SIXLOWPAN_FRAG_RBUF_TIMEOUT_US` -> `CONFIG_GNRC_SIXLOWPAN_FRAG_RBUF_TIMEOUT_US`.